apps/generator/docs/generator-template.md

```
@@ -83,19 +83,21 @@ components:
           type: string
 ```
 
-## Overview of steps
+ ## Overview of steps
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)